circuits/circom/tallyVotesNonQv.circom

```diff
@@ -184,13 +170,95 @@ template TallyVotesNonQv(
     rcv.currentSpentVoiceCreditSubtotalSalt <== currentSpentVoiceCreditSubtotalSalt;
     rcv.newSpentVoiceCreditSubtotal <== newSpentVoiceCreditSubtotal.sum;
     rcv.newSpentVoiceCreditSubtotalSalt <== newSpentVoiceCreditSubtotalSalt;
-    rcv.currentPerVOSpentVoiceCreditsRootSalt <== currentPerVOSpentVoiceCreditsRootSalt;
-    rcv.newPerVOSpentVoiceCreditsRootSalt <== newPerVOSpentVoiceCreditsRootSalt;
 
     for (var i = 0; i < numVoteOptions; i++) {
         rcv.currentResults[i] <== currentResults[i];
         rcv.newResults[i] <== resultCalc[i].sum;
-        rcv.currentPerVOSpentVoiceCredits[i] <== currentPerVOSpentVoiceCredits[i];
-        rcv.newPerVOSpentVoiceCredits[i] <== newPerVOSpentVoiceCredits[i].sum;
     }
 }
+
+// Verifies the commitment to the current results. Also computes and outputs a
+// commitment to the new results. Works for non quadratic voting 
+// - so no need for perVOSpentCredits as they would just match 
+// the results.
+template ResultCommitmentNonQvVerifier(voteOptionTreeDepth) {
```

Comment:
Can we move this template to a separate file?

Reply:
could we keep as is for now? there's another branch with a WIP cleanup of this folder and re organization https://github.com/privacy-scaling-explorations/maci/tree/refactor/circom-circuits

Reply:
Sure, it works